src/main/java/frc/robot/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -91,7 +91,7 @@ public Command setExtensionCmd(DoubleSupplier meters) {
 
   public Command runCurrentZeroing() {
     return this.run(() -> io.setVoltage(-1.0))
-        .until(() -> inputs.elevatorCurrentAmps[0] > 40.0)
+        .until(() -> inputs.elevatorStatorCurrentAmps[0] > 40.0)
```

Comment:
this syntax is very scary for me. Not sure if this is the syntax or norm for Java, but readers can be unsure of the value difference between index 0, 1, ... (even how many there are? Abstracting to a class like mentioned above should clear up your intentions). I always repeat the phrase: write once, read 10 times.

Reply:
also see this line being repeated in `src/main/java/frc/robot/subsystems/shooter/ShooterSubystem.java` :151, but with a different syntax of using the array. maybe clearing some of this stuff up might help clear confusion among fresh eyes.

Reply:
Yeah, I agree. Seems like a good use for a record. Additionally, the value appears to be initialized to a 0 length array but then we do array access here that would be out of bounds if the value isn't set to something else. If the size of the array changes it strongly implies that maybe array isn't the right container.

Reply:
the whole array use was a holdover from the akit examples, where they wanted you to use an array for all motor current and temperature since they consider motor count an implementation detail. i dont really agree w their view of it either, and will change it